h/panels/navbar.py

```diff
@@ -29,14 +30,15 @@ def navbar(context, request, search=None, opts=None):
                 'title': group.name,
                 'link': request.route_url('group_read', pubid=group.pubid, slug=group.slug)
             })
-            groups_suggestions.append({
```

Comment:
What is the plan regarding showing associated groups in the groups drop-down menu?